application/sbsk-web/src/main/java/com/sbsk/web/logging/appender/FluentAppender.java

```diff
@@ -15,17 +15,29 @@
 
 public class FluentAppender extends AppenderBase<ILoggingEvent> {
 
+  private static final String HOST_KEY = "host";
+  private static final String APP_KEY = "app";
+  private static final String ENV_KEY = "environment";
+  private static final String MAIN_KEY = "main";
+  private static final String THREAD_KEY = "thread";
+  private static final String LEVEL_KEY = "level";
+  private static final String CLASS_KEY = "class";
+  private static final String TYPE_KEY = "type";
+  private static final String EXCEPTIONS_KEY = "exceptions";
+  private static final String MESSAGE_KEY = "message";
+  private static final String STACKTRACE_KEY = "stacktrace";
+  private static final String TYPE_EXCEPTION = "EXCEPTION";
+  private static final String TYPE_MESSAGE = "MESSAGE";
+
```

Comment:
because there are too many Constants, you could move them to another class like 'FluentAppenderConstants'